internal/server/handlers.go

```diff
@@ -401,11 +401,18 @@ func (a *API) ListAccessKeys(c *gin.Context, r *api.ListAccessKeysRequest) ([]ap
 	results := make([]api.AccessKey, len(accessKeys))
 
 	for i, a := range accessKeys {
+		// TODO: this would be better as a JOIN query
+		identity, err := access.GetIdentity(c, a.IssuedFor)
```

Comment:
This is still iterating and querying once for ever key, which is not ideal.

The way I am used to doing this would be to instead have a join query between access keys and identities, so that we only need one query. I'm not sure how to do that with GORM.